client/my-sites/theme/style.scss

```diff
@@ -20,6 +20,28 @@ body.is-section-theme-i4 {
 		}
 	}
 
+	.theme__sheet-action-bar-container {
+		box-shadow: 0 0 0 1px var(--color-border-subtle);
```

Comment:
It's a minor issue but I assume `0 1px var(--color-border-subtle)` is enough or we can simply use `border-bottom`. Otherwise, you will see the box shadow on the top

![image](https://user-images.githubusercontent.com/13596067/223639264-596933d0-6597-48ca-a1fb-58bf272656e8.png)